src/dials/command_line/correlation_matrix.py

```diff
@@ -23,11 +23,14 @@
 
 phil_scope = iotbx.phil.parse(
     """\
-include scope dials.algorithms.correlation.analysis.phil_scope
+include scope dials.algorithms.correlation.analysis.working_phil
 
 seed = 42
   .type = int(value_min=0)
 
+outlier_rejection = True
```

Comment:
For clarity it might be better to nest this parameter, alongside a parameter for the maximum number of dimensions to test, into a scope with a name such as 'dimensionality assessment'.

Reply:
Have nested this as well as maximum_dimensions into dimensionality_assessment parameter